src/utils/printName.ts

```diff
@@ -41,14 +41,14 @@ function printArrayName(schemaNode: RegularNode): string | null {
         return mergedTypes;
       }, []) ?? null;
 
-    return val !== null && val.length > 0 ? `${SchemaNodeKind.Array}[${val.join(',')}]` : null;
+    return val !== null && val.length > 0 ? `array of ${val.join('s/')}s` : null;
   }
 
   if (isComplexArray(schemaNode)) {
     const firstChild = schemaNode.children[0];
-    return firstChild.title !== null
-      ? `${firstChild.title}[]`
-      : `${SchemaNodeKind.Array}[${firstChild.primaryType ?? firstChild.combiners?.join(',')}]`;
+    return firstChild.title
+      ? `array of ${firstChild.title}-s`
```

Comment:
Did you consider using https://github.com/plurals/pluralize?
Although I'm usually not a huge fan of adding libraries, this one might turn out to be useful.
Adding `s` by default won't be valid in all cases, i.e.
`watch` -> `watches`, `witch` -> `witches`, or `water`, `rice`, etc.

Reply:
I did consider that exact library, but I realized I don't need it here:
- If it's an object name (like in this case), I much prefer to have the hyphen, because it emphasizes that those are entities with a name. Let's say I have an object titled `Integer` (for some effed up reason). If it says *array of Integer-s*, it's clear that it's different from e.g. *array of numbers*.
- For primitives, fortunately a simple `s` fits them all.
- For objects, I really don't want the English rules. Think about an object called *Person*. I don't want the array to say *array of People* 😄 I agree re *array of Watch-s* isn't ideal but it gets the point across, and that case would be hard to cover even with the library (it doesn't do hyphenation). I'd say that's an improvement for later.